src/compas_fab/ghpython/components/__init__.py

```diff
@@ -1,5 +1,62 @@
 from __future__ import absolute_import
 
+try:
+    import Grasshopper  # type: ignore
+except ImportError:
+    pass
+
 
 def create_id(component, name):
     return "{}_{}".format(name, component.InstanceGuid)
```

Comment:
This method is actually now on [`compas_ghpython`](https://github.com/compas-dev/compas/blob/main/src/compas_ghpython/__init__.py), so we could remove and replace for that one.